### PR TITLE
Update ann data notebook

### PR DESCRIPTION
**Pull request by PozhidayevaDarya**
Tables in notebook changed to MSK from CHOP.

### CODE REVIEW

File: HTAN/Python Notebooks/Building_AnnData_with_Subset_of_Cells_from_BQ.ipynb [hunk 1]
```diff
@@ -0,0 +1,1312 @@
...
+      "execution_count": 7,
+      "metadata": {
+        "id": "6AiGLpeFGfH3"
+      },
+      "outputs": [],
+      "source": [
+        "## Setup ##\n",
+        "\n",
+        "# set the google project that will be billed for this notebook's computations\n",
+        "google_project = 'htan-dcc'\n",
```
Comment (clarisse-lau) [inline HTAN/Python Notebooks/Building_AnnData_with_Subset_of_Cells_from_BQ.ipynb]: For consistency with other notebooks, generalize project name?

**Review by vthorsson — COMMENTED** [review]
Minor warnings appear on running 
# now we can build the AnnData
(variable names not unique)
and 
# now we can use the plot functions of scanpy
(No data for colormapping provided)

Also we should change the specific google project name to 'my-project' or some such, as we do elswhere

**Comment by PozhidayevaDarya** [comment]
> Minor warnings appear on running
> 
> # now we can build the AnnData
> (variable names not unique) and
> 
> # now we can use the plot functions of scanpy
> (No data for colormapping provided)
> 
> Also we should change the specific google project name to 'my-project' or some such, as we do elswhere




For the warnings should I just hide them or do they need to be deduplicated? @Gibbsdavidl do you know?

**Comment by clarisse-lau** [comment]
Thanks for the edits @PozhidayevaDarya and @Gibbsdavidl :)

Just have a few more questions/suggestions:
- Another specific Google project name was introduced in the most recent commit. Reiterating comments above to change to something generic like `my-project`
- Is matplotlib used in the notebook (or is it just the scanpy plot functions)? If the latter, the import section can be removed
- There are still a few warnings showing up in the output - do we plan to leave as is?

**Comment by PozhidayevaDarya** [comment]
> Thanks for the edits @PozhidayevaDarya and @Gibbsdavidl :)
> 
> Just have a few more questions/suggestions:
> 
> * Another specific Google project name was introduced in the most recent commit. Reiterating comments above to change to something generic like `my-project`
> * Is matplotlib used in the notebook (or is it just the scanpy plot functions)? If the latter, the import section can be removed
> * There are still a few warnings showing up in the output - do we plan to leave as is?

I think the warnings will just persist so i'm not sure we can get rid of them but I went ahead and changed the other suggestions. Thanks! 

@Gibbsdavidl lemme know what you think too.

**Review by clarisse-lau — APPROVED** [review]

**Comment by Gibbsdavidl** [comment]
Yeah, wouldn't worry about the warnings.  They seem to pop up as the software changes.

**Pull request merged**